README.md

```diff
@@ -24,6 +24,7 @@ Specifically, the responsibilities of this role are to:
 | `openio_oioswift_filter_*` | `dict` | Options of most used middleware |
 | `openio_oioswift_gridinit_dir` | `/etc/gridinit.d/{{ openio_oioswift_namespace }}` | Path to copy the gridinit conf |
 | `openio_oioswift_gridinit_file_prefix` | `""` | Maybe set it to `{{ openio_oioswift_namespace }}-` for legacy gridinit's style |
+| `openio_oioswift_inventory_groupname` | `""` | Set your inventory groupname to auto feed memcached server on port `6019` |
```

Comment:
Could you speak about `default port` here, so that when we variabilize it we won't have to change this doc again